powersimdata/scenario/create.py

```diff
@@ -251,6 +201,10 @@ def set_builder(self, grid_model="usa_tamu", interconnect="USA"):
         self._scenario_info["grid_model"] = self.builder.grid_model
         self._scenario_info["interconnect"] = self.builder.interconnect
 
+    def set_grid(self, *args, **kwargs):
+        """Alias to set_builder."""
+        self.set_builder(*args, **kwargs)
```

Comment:
Should we have a `DeprecationWarning` here to incite users to use this method and not `set_builder`? The idea would be to remove `set_builder` in a future release.

Reply:
If we want to do that, then we should move the functional code to `set_grid` and have `set_builder` be the alias with the warning.

Reply:
Done.